mhw_analysis/ncep/io.py

```diff
@@ -0,0 +1,51 @@
+""" Module to deal with I/O of CEMS-LENS files"""
+import os
+
+import numpy as np
+
+import iris
+
+
+def load_z500(dmy, path=None, end_dmy=None):
```

Comment:
might be good to keep a default path, or environmental variable

Reply:
done.